integration/compiler-test.ts

```diff
@@ -36,8 +36,30 @@ describe("compiler", () => {
           import * as path from "path";
 
           export default function BuiltIns() {
-            return <div id="built-ins-polyfill">{path.join("test", "file.txt")}</div>
+            return <div id="built-ins-polyfill">{path.join("test", "file.txt")}</div>;
           }
+        `,
+        "app/routes/esm-only-pkg.jsx": js`
+          import esmOnlyPkg from "esm-only-pkg";
+
+          export default function EsmOnlyPkg() {
+            return <div id="esm-only-pkg">{esmOnlyPkg}</div>;
+          }
+        `,
+        "remix.config.js": js`
+          module.exports = {
+            ignoredRouteFiles: [".*"],
```

Comment:
```suggestion
```

Better to leave unrelated things out to avoid people thinking it's a relevant part of the test.